src/core/ctx.ts

```diff
@@ -163,7 +163,7 @@ ${dts}`.trim()}\n`
 
   async function parseESLint() {
     const configStr = existsSync(eslintrc.filepath!) ? await fs.readFile(eslintrc.filepath!, 'utf-8') : ''
-    const config = JSON.parse(configStr || '{ "globals": {} }')
+    const config = !eslintrc.filepath?.endsWith('json') ? JSON.parse(configStr.replace('export default', '').replace('module.exports =', '') || '{ "globals": {} }') : JSON.parse(configStr || '{ "globals": {} }')
```

Comment:
This is not a safe assumption. I'd say we ignore parsing in non-json mode.

Reply:
In my case.I can't use JSON format files in my `eslint.config.js` file. So, I'll go with the CJS format for my setup. Our plugin works great when exporting JSON, but as soon as I switch to CJS, it runs into problems and crashes.

Even though it doesn't affect my code running, it does cause ESLint to throw errors.

env: 
"eslint": "^9.3.0",

```
export default [
  {
    files: ['**/*.{js,vue}'],
    languageOptions: {
      globals: {
        ...globals.browser,
        ...globals.node,
        ...autoImportConfig.globals,     <-------
      },
      parser: vueEslintParser,
      parserOptions: {
        ecmaVersion: 2020,
        sourceType: 'module',
      },
    },
    plugins: {
      vue: pluginVue,
      prettier: prettierPlugin,
    },
    settings: {},
    rules: {
      ...pluginJs.configs.recommended.rules,
      ...pluginVue.configs['flat/essential'].rules,
      'prettier/prettier': 'error',
    },
  },
  {
    rules: {
      ...prettier.rules,
    },
  },
];
```